packages/fxa-settings/src/pages/Signin/Signin/index.tsx

```diff
@@ -0,0 +1,248 @@
+                        className="inline"
+                        aria-label={localizedLogoAriaLabel}
+                      />
+                    </span>
+                  </span>
+                </FtlMsg>
+              )}
+              {!isPocketClient && !serviceLogo && (
+                <FtlMsg
+                  id="signin-subheader-without-logo"
+                  vars={{ serviceName }}
+                >
+                  <span className="card-subheader">
+                    Continue to {serviceName}
+                  </span>
+                </FtlMsg>
+              )}
+            </>
+          </>
+        )}
+      </div>
+      <section>
+        {/* Alerts and success messages originally went here */}
+        <div className="mt-9">
+          <div className="avatar-wrapper mx-auto h-24 w-24 mobileLandscape:h-40 mobileLandscape:w-40"></div>
+          <div className="my-5 text-base break-all">{email}</div>
+        </div>
+        <form noValidate {...{ onSubmit }}>
+          <input type="email" className="email hidden" value={email} disabled />
+
+          {isPasswordNeeded && (
+            <InputPassword
+              anchorStart={true}
+              className="mb-2 text-start"
+              label={localizedPasswordFormLabel}
+              hasErrors={error.length > 0}
+              errorText={error}
+              tooltipPosition="bottom"
+              required={true}
+              autoFocus={true}
+              onChange={(e) => setPassword(e.currentTarget.value)}
+            />
+          )}
+          {/* This non-fulfilled input tricks the browser, when trying to
+              sign in with the wrong password, into not showing the doorhanger.
+           */}
+          <input className="hidden" required />
+
+          <div className="flex">
+            <FtlMsg id="signin-button">
+              <button className="cta-primary cta-xl" type="submit">
+                Sign in
+              </button>
+            </FtlMsg>
+          </div>
+        </form>
+        <div id="tos-pp" className="text-grey-500 my-5 text-xs">
+          {isPocketClient ? (
+            <FtlMsg id="firefox-and-pocket-tos">
+              <p className="text-xs">
+                By proceeding, you agree to:
+                <br />
+                Pocket’s{' '}
+                <LinkExternal
+                  className="link-grey"
+                  href="https://getpocket.com/tos/"
+                >
+                  Terms of Service
+                </LinkExternal>{' '}
+                and{' '}
+                <LinkExternal
+                  className="link-grey"
+                  href="https://getpocket.com/privacy/"
+                >
+                  Privacy Notice
+                </LinkExternal>
+                <br />
+                Firefox’s{' '}
+                <LinkExternal className="link-grey" href="/legal/terms">
+                  Terms of Service
+                </LinkExternal>{' '}
```

Comment:
I don't think ToS and privacy notice should use `LinkExternal` since they'll link to a page in the React app. 

Also it miiight be worth creating a component for this called `LinkTermsOfService` and same for privacy notice either here or in the near future, I think we use these on other pages too, but 🤷‍♀️

Reply:
Oh darn, this is totally my fault, I hadn't realized that these link to pages within our app (vs terms from the footer). I'll go ahead and fix that in my Signup PR too.

@millsoper your PR is closer to landing than mine, I could pull this out into its own component from the Signup component and update the Signin page once yours land, if that works for you?